kolibri/plugins/registry.py

```diff
@@ -68,7 +68,17 @@ def initialize():
 
         for app in settings.INSTALLED_APPS:
             try:
-                plugin_module = importlib.import_module(app + ".kolibri_plugin")
+
+                # Handle AppConfig INSTALLED_APPS string
+                if ".apps." in app:
+                    # remove .apps.Config line in string
+                    import_string = app.split('.apps.')[0]
+                else:
+                    import_string = app
+
+                import_string += ".kolibri_plugin"
+                plugin_module = importlib.import_module(import_string)
```

Comment:
Yet another reason to have a list of *plugins* rather than injecting stuff straight into `INSTALLED_APPS` (same conclusion I came to when detecting if a builtin plugin exists or not)